### ISSUE TITLE
Improving the borders

### ISSUE THREAD

**Issue by daelba** [Open-Historical-Map-Labs/openhistoricaltiles#51]
Many thanks for improving the borders. However, it does not render admin_level=3 (e.g. http://www.openhistoricalmap.org/relation/2661870). Compare with timeslider: https://openhistoricalmap.github.io/openhistoricaltiles/mbgl-control-timeslider/demo/#7.423/49.75295/17.27063/1905,1850-1945.

┆Created By: Dan Rademacher

**Comment by jeffreyameyer** [comment]
Daniel - good catch - thank you!

@mojodna - any thoughts?

**Comment by sethfitz** [comment]
This definitely looks like a stylesheet issue (inherited from OpenMapTiles); the data is present, but there's this note in the boundary config:

https://github.com/OpenHistoricalMap/openmaptiles/blob/3710bbcddc5dea932302c8049cb73969891916b7/layers/boundary/boundary.yaml#L4-L6

> Contains administrative boundaries (no maritime boundaries yet) as polygons.
> OSM data contains all [`admin_level`](http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dadministrative#admin_level)
> but for most styles it makes sense to just style `admin_level=2` and `admin_level=4`.

**Comment by daelba** [comment]
I have structured the levels in this way:
admin_level=2 country
admin_level=3 Crown land
admin_level=4 political district
admin_level=6 court district
admin_level=8 municipality
admin_level=10 cadastral unit

I have tried to shift 3-4-6 to 4-5-6/7 before, however didn't work well, because the borders were rendered first at too high zoom level. That means, that it wasn't possible to see the whole Crown land / political district and all its subdivisions, because it had to be zoomed too much to see the subdivision borders.

So, I see two solutions:
1. change the code to render admin_level=3 or
2. change admin_level=3-4-6 to 4-5-6/7 and also change the code to render admin_level=3/4/5 sooner (at lower zoom).

**Comment by tsinn** [comment]
Added in admin level 3 to GIN's Leaflet map. Can be improved in the future. Currently rendering same as admin levels 4-8.

**Comment by daelba** [comment]
`admin_level=3` is not rendered again. Can it be fixed?

**Comment by jeffreyameyer** [comment]
Yes, we have a stylesheet bug issue right now. We're working on it!

**Comment by tsinn** [comment]
Hi there, @daelba! I'm working through styling issues. Could you point me to an example location where `admin_level=3` is not rendered? Thank you!

**Comment by daelba** [comment]
Hi, for example https://www.openhistoricalmap.org/relation/2664912#map=11/48.9728/15.0760&layers=O.
